resources/js/Pages/Dashboard.vue

```diff
@@ -74,9 +75,17 @@ import { ref } from 'vue';
                                 </PrimaryButton>
                             </div>
                         </div><!-- post-item end -->
-                        <PrimaryButton v-if="this.posts.length>3 && this.postLength < this.posts.length" @click="loadMore" class="offset-lg-5 col-md-3">
-                            Load More
-                        </PrimaryButton>
+                        <div>
+                            <a  href="javascript:void(0)" @click="loadMorePosts(link.url)">{{ index }}</a>
+                        </div>
+                        <!-- pagination -->
+                        <nav aria-label="Page">
+                            <ul class="pagination float-end">
+                                <li v-for="(link,index) in loadMoreLinks"  class="page-item">
+                                    <a v-bind:class="{ active: link.active }" class="page-link" @click="loadMorePosts(link.url)" v-if="index>0 && index<loadMoreLinks.length-1" href="javascript:void(0)">{{ index }}</a>
```

Comment:
You don't need to type `v-bind` in this instance. Just typing `:class` works the same way.